ros2bag/ros2bag/verb/record.py

```diff
@@ -52,6 +51,13 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='time in ms to wait between querying available topics for recording. It has no '
              'effect if --no-discovery is enabled.'
         )
+        parser.add_argument(
+            '-b', '--bag-size', type=int, default=sys.maxsize,
```

Comment:
What are the units of the size?

Reply:
It's in `bytes`, I'll add that in the `help` section.

Reply:
nit: I'd include the size in the variable name so intent is clear in code as well.